src/ccall.cpp

```diff
@@ -1270,21 +1270,21 @@ static jl_cgval_t emit_ccall(jl_value_t **args, size_t nargs, jl_codectx_t *ctx)
                 if (jl_is_expr(args[2])) {
                     jl_expr_t *rtexpr = (jl_expr_t*)args[2];
                     if (rtexpr->head == call_sym && jl_expr_nargs(rtexpr) == 4 &&
-                            static_eval(jl_exprarg(rtexpr, 0), ctx, true, false) == jl_builtin_apply_type &&
-                            static_eval(jl_exprarg(rtexpr, 1), ctx, true, false) == (jl_value_t*)jl_array_type) {
+                        static_eval(jl_exprarg(rtexpr, 0), ctx, true, false) == jl_builtin_apply_type &&
+                        static_eval(jl_exprarg(rtexpr, 1), ctx, true, false) == (jl_value_t*)jl_array_type) {
```

Comment:
the elseif's are indented in a clearer way than the ifs with this. it's pretty divided, we should probably pick a single preference.